meta.go

```diff
@@ -194,6 +194,7 @@ type Meta struct {
 	Series         []string                `bson:"series,omitempty" json:"supported-series,omitempty"`
 	Storage        map[string]Storage      `bson:"storage,omitempty" json:"storage,omitempty"`
 	PayloadClasses map[string]PayloadClass `bson:"payloadclasses,omitempty" json:"payloadclasses,omitempty"`
+	Terms          []string                `bson:"terms,omitempty" json:"terms,omitempty`
```

Comment:
This could do with documenting.
Also, as discussed on-line, I think "Conditions" would be a more appropriate name here, as "Terms" is highly ambiguous in a computer context. TermsAndConditions would also be OK, but Conditions is more general and perhaps usefully cross-applicable to a number of future situations.